clefos/content.md

```diff
@@ -0,0 +1,98 @@
+
+%%LOGO%%
+
+## ClefOS image documentation
+
+The `clefos:latest` tag is always the most recent version currently available.
+
+### Building the Base Image
+
+The image is built via the `make` command which will create the tarball, build the image, and push the image to the repository.
+
+The `createBase.sh` script is used to create the tarball for the docker build command. The script uses the yum command with the `tsflags=nodocs` option set to reduce the size of the image. In addition a lot of the locale files are eliminated from the image.
+
+The `VERSION` file contains the id of the current ClefOS version and will be added as a label within the image.
+
+### Rolling builds
+
+The ClefOS Project offers regularly updated images for all active releases. These images will be updated monthly or as needed for emergency fixes. These rolling updates are tagged with the major version number only. For example: `docker pull clefos:7`
+
+### Minor tags
+
+Additionally, images with minor version tags that correspond to install media are also offered. **These images DO NOT recieve updates** as they are intended to match installation iso contents. If you choose to use these images it is highly recommended that you include `RUN yum -y update && yum clean all` in your Dockerfile, or otherwise address any potential security concerns. To use these images, please specify the minor version tag:
+
+For example: `docker pull clefos:7.3.1611`
+
+### Overlayfs and yum
+
+Recent Docker versions support the [overlayfs](https://docs.docker.com/engine/userguide/storagedriver/overlayfs-driver/) backend, which is enabled by default on most distros supporting it from Docker 1.13 onwards. On ClefOS 7, **that backend requires yum-plugin-ovl to be installed and enabled**; while it is installed by default in recent clefos images, make it sure you retain the `plugins=1` option in `/etc/yum.conf` if you update that file; otherwise, you may encounter errors related to rpmdb checksum failure - see [Docker ticket 10180](https://github.com/docker/docker/issues/10180) for more details.
+
+## Package documentation
+
+By default, the ClefOS containers are built using yum's `nodocs` option, which helps reduce the size of the image. If you install a package and discover files missing, please comment out the line `tsflags=nodocs` in `/etc/yum.conf` and reinstall your package.
+
+## Systemd integration
+
+Systemd is now included in both the clefos:7 and clefos:latest base containers, but it is not active by default. In order to use systemd, you will need to include text similar to the example Dockerfile below:
```

Comment:
This doesn't apply at all to your images, right?  I seem to recall `systemd` being one of the major things removed from yours vs `centos:latest` (which IMO is a good thing, since it's not normally necessary in containers).